modules/mw.cx.TranslationMdwiki.js

```
@@ -43,98 +67,118 @@ async function fetchGetJson(url) {
+		revision: "5200",
+		segmentedContent: "",
+		categories: []
 	}
+	// ---
+	out.segmentedContent = removeUnlinkedWikibase(data.html);
+	// ---
+	const html = data.html.replaceAll("&#34;", '"');
+	const matches = html.match(/Mdwiki_revid"\},"params":\{"1":\{"wt":"(\d+)"\}\}/);
+	// ---
+	if (matches && matches[1]) {
+		out.revision = matches[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)